<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCSL2_AskDialog</class>
 <widget class="QDialog" name="MCSL2_AskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>提示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/MCSL2_Icon.qrc">
    <normaloff>:/MCSL2_Icon/MCSL2_Icon.png</normaloff>
    <normalon>:/MCSL2_Icon/MCSL2_Icon.png</normalon>
    <disabledoff>:/MCSL2_Icon/MCSL2_Icon.png</disabledoff>
    <disabledon>:/MCSL2_Icon/MCSL2_Icon.png</disabledon>
    <activeoff>:/MCSL2_Icon/MCSL2_Icon.png</activeoff>
    <activeon>:/MCSL2_Icon/MCSL2_Icon.png</activeon>
    <selectedoff>:/MCSL2_Icon/MCSL2_Icon.png</selectedoff>
    <selectedon>:/MCSL2_Icon/MCSL2_Icon.png</selectedon>:/MCSL2_Icon/MCSL2_Icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgba(247, 247, 247,85%);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="Dialog_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>151</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	background-color: rgb(255, 255, 255);
	border-radius: 10px
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="ButtonWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>35</horstretch>
       <verstretch>40</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>351</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>351</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Dialog_PushButton_Cancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: rgb(255, 255, 255);
	border-radius: 7px;
}
QPushButton:hover
{
	background-color: rgb(240, 240, 240);
	border-radius: 7px;
}
QPushButton:pressed
{
	background-color: rgb(229, 229, 229);
	border-radius: 7px;
}</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Dialog_PushButton_Accept">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background-color: rgb(0, 120, 212);
	border-radius: 7px;
	color: rgb(255, 255, 255);
}
QPushButton:hover
{
    background-color: rgb(0, 110, 212);
	border-radius: 7px;
    color: rgb(255, 255, 255);
}
QPushButton:pressed
{
    background-color: rgb(0, 100, 212);
	border-radius: 7px;
    color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons/MCSL2_Icon.qrc"/>
 </resources>
 <connections/>
</ui>
